docs/src/main/asciidoc/_admin-guide/management-tasks/YAML_Configuration_file.adoc

```diff
@@ -1,19 +1,17 @@
 [[YAML_Configuration_file]]
 = (YAML) Configuration file (Experimental))
 
-In order to provide some way to separate customization from the configuration provisionned by Galleon we want to *experiment* with a new feature where this customization is provided by YAML files.
-Because this feature is *EXPERIMENTAL* we are disabling it by default and you can *NOT* rely on it being present or compatible in the future releases.
+Provides a way to separate customization from the configuration provisioned by Galleon by using YAML configuration files.
+Because this feature is *EXPERIMENTAL* it may removed in future releases.
 
-[[activate_yaml_feature]]
-== Active the YAML support feature
+[[disable_yaml_feature]]
+== Disable the YAML support feature
 
-To enable that feature you need to add a ServiceLoader configuration in the `org.jboss.as.controller` module.
-You need to create the following file: `META-INF/services/org.jboss.as.controller.persistence.ConfigurationExtension` containing a single line *org.jboss.as.controller.persistence.yaml.YamlConfigurationExtension* in the dir folder of the `org.jboss.as.controller` module.
+To disable the YAML configuration feature, remove the ServiceLoader configuration in the `org.jboss.as.controller` module by removing file: `META-INF/services/org.jboss.as.controller.persistence.ConfigurationExtension`.  The removed file contains a single line *org.jboss.as.controller.persistence.yaml.YamlConfigurationExtension* in the dir folder of the `org.jboss.as.controller` module.  
```

Comment:
We don't support disabling the feature.

Reply:
Updated to remove information on disabling.